docs/docs/platform/templates.md

```diff
@@ -6,33 +6,33 @@ sidebar_position: 1
 
 A template holds the entire flow of messages sent to the subscriber. This is where all the different channels are tied together under a single entity.
 
-Let's explore the different parts of a notification template
+Let's explore the different parts of a notification template.
 
 ## Global template metadata
 
 This will contain general information regarding the notification itself. Let's explore the different aspects of it.
 
 ### Notification name
 
-The name will be used to identify the notification template when triggering it. A slugified version of the name will be generated after the notification was created. For example, a notification template with the name of "Test Notification" will be converted to "test-notification" as the trigger key.
+The name used to identify the notification template when triggering it. A slugified version of the name will be generated after the notification was created. For example, a notification template with the name of "Test Notification" will be converted to "test-notification" as the trigger key.
 
 ### Notification Group
 
-The notification group is used to group multiple notification templates into a single group, currently only used behind the scenes for organisational purposes. But, in the upcoming subscriber preferences management, it will be used to group multiple notifications for the subscriber.
+The notification group is used to group multiple notification templates into a single group (currently only used behind the scenes for organizational purposes). But, in the upcoming subscriber preferences management, it will be used to group multiple notifications for the subscriber.
```

Comment:
I am sorry but this copy doesn't provide improvements to be honest.
`organisational` is rightly spelled (british english). It is undecided an unique language variaton for the whole codebase.
@scopsy maybe something to agree and to add to the CONTRIBUTING.md file.